### PR TITLE
Make relink.script more resilient

### PR DESCRIPTION
**Pull request by rbuch**
If one of subdirs was missing, rm would fail to remove any of
them, so now rm each member of subdirs individually.

### CODE REVIEW

File: relink.script [hunk 1]
```diff
@@ -23,12 +23,11 @@ fi
 
 subdirs="bin include lib lib_so tmp"
 
-# remove symbolic link
-rm -f "$subdirs"
```
Comment (matthiasdiener) [inline relink.script:27]: I think it is easier to just remove the `"` from here. This is a mistake introduced by #2951 .

Comment (rbuch) [reply relink.script:27]: Done.

Comment (rbuch) [reply relink.script:27]: It seems like doing this causes it to fail the ShellCheck test. Is there a simple way around this?

Comment (matthiasdiener) [reply relink.script:27]: Can you do the same fix as in https://github.com/UIUC-PPL/charm/commit/c30789cae98a71582c20edf10fc29b3497d3fdf2 ? I'm without internet at the moment, so I can't add this myself.

**Pull request merged**